Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -368,4 +372,32 @@ extension ChatChannelListItemView {
     var isLastMessageVoiceRecording: Bool {
         content?.channel.previewMessage?.voiceRecordingAttachments.isEmpty == false && typingUserString == nil
     }
+
+    /// The message preview text in case it contains attachments.
+    /// - Parameter previewMessage: The preview message of the channel.
+    /// - Returns: A string representing the message preview text.
+    func attachmentPreviewText(for previewMessage: ChatMessage) -> String? {
```

Comment:
Does it make sense for this to be customizable by customers?

Reply:
Yeah, I think it kind of does. They can do it anyway if they override the subtitle text, and it should be easy as well. For that reason, I didn't open it up yet. My plan was to keep it internal for now, and if a customer really needs it, we can make it open. What do you think?